### PR TITLE
Fix a mistake in the team article

### PR DESCRIPTION
**Pull request by sprunk**
Well whaddya know, there actually is native support for factions

### CODE REVIEW

File: doc/site/articles/team-terminology.markdown [hunk 1]
```diff
@@ -24,7 +24,8 @@ For modders coming from Supreme Commander, the *team* is what SupCom calls an "a
  * resources
  * colour
  * starting point
- * extra traits, such as faction (the engine does not natively recognize the concept of factions but it's easy to add them)
+ * faction (aka side)
+ * any other custom extra traits you add.
```
Comment (sprunk) [inline doc/site/articles/team-terminology.markdown:28]: I couldn't think of a good universal example of something that could be Lua'd which is obvious without too much explanation.

Comment (badosu) [reply doc/site/articles/team-terminology.markdown:28]: Where is the concept of faction encoded in the engine?

**Comment by badosu** [comment]
LGTM! :+1:

File: doc/site/articles/team-terminology.markdown (hunk 1 above)
Comment (sprunk) [reply doc/site/articles/team-terminology.markdown:28]: Yes, "side".

**Pull request merged**